server/routes/notification.js

```diff
@@ -82,6 +84,79 @@ router.get('/manage', async (req, res, next) => {
   } catch (error) {
     return next(error)
   }
-})
+}
+router.get('/manage', getManageMiddleware)
```

Comment:
would have been nicer to put this in a new file and could then have written it in typescript, rather than writing more javascript.

Reply:
This gets complicated ! Leaving for now